x-pack/solutions/search/test/api_integration/apis/search_playground/playground_utility_apis.ts

```diff
@@ -0,0 +1,333 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from 'expect';
+import { ELASTIC_HTTP_VERSION_HEADER } from '@kbn/core-http-common';
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { ALL_USERS, USERS } from './common/users';
+import { ALL_ROLES } from './common/roles';
+import { createUsersAndRoles, deleteUsersAndRoles } from './common/helpers';
+
+const INITIAL_REST_VERSION = '1' as const;
```

Comment:
you shouldn't need this for these endpoints, only the saved playground routes are currently using versions.